app/tasks.py

```
@@ -0,0 +1,122 @@
+    await asyncio.sleep(0.03 + random.random() * 0.05)
+
+    # Random transient failure (e.g., network, DB lock)
+    if random.random() < 0.12:
+        raise RuntimeError("transient failure")
+
+    # Increment *attempt* count, not unique processing count (B1)
+    _processed[event_id] = _processed.get(event_id, 0) + 1
+    return _processed[event_id]
+
+
+async def worker(name: str = "worker-1", max_retries: int = 3) -> None:
```

Ruff (0.12.2)

Unused function argument: `name`

(ARG001)